tests/x11/addon_products_via_SCC_yast2.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: add addon to SLES via SCC
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base qw(y2_installbase y2_module_guitest);
+use strict;
+use warnings;
+use testapi;
+use registration;
+use version_utils qw(is_sle is_leap);
+use x11utils 'turn_off_gnome_screensaver';
+use YaST::Module;
+use utils;
+
+sub test_setup {
+    select_console "x11";
+    x11_start_program('xterm');
+    turn_off_gnome_screensaver;
+}
+
+sub run {
+    my $self = shift;
+    test_setup;
+    YaST::Module::open(module => 'scc', ui => 'qt');
+    save_screenshot;
+    $testapi::distri->get_registration()->register_via_scc({
+            email => get_var('SCC_EMAIL'),
+            reg_code => get_var('SCC_REGCODE')});
+    save_screenshot;
+    my @scc_addons = split ',', get_var('SCC_ADDONS');
+    $testapi::distri->get_module_registration()->register_extension_and_modules([@scc_addons]);
+    save_screenshot;
+
+    # No libyui-rest-api for advance software selection
+    assert_screen("yast_scc-pkgtoinstall", 100);
+    wait_screen_change {
```

Comment:
(optional) we can do one line here.

Reply:
This format copy form open.qa document.

Reply:
both ways are ok, one just save lines of code, code style, just less to read, shortest test. But it is ok as it is, just consider the length of the test file in the future, comparing with the scary files that we can see for legacy with needles and shortcuts.